thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1565,6 +1568,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
             CAsyncCallStart asyncLeftStart(std::bind(&CInMemJoinBase::startLeftInput, this));
             try
             {
+                ActivityTimer timer(slaveTimerStats, timeActivities);
```

Comment:
should be a head of start, so it includes the whole of start() time afaics.
and normally for convention only: "ActivityTimer s(slaveTimerStats, timeActivities);" i.e. 's' for start.